includes/wikia/models/DesignSystemGlobalNavigationModel.class.php

```diff
@@ -382,4 +402,23 @@ private function getSitenameData() {
 			'value' => WikiFactory::getVarValueByName( 'wgSitename', $this->productInstanceId, false, $this->wg->Sitename ),
 		];
 	}
+
+	private function getRightSlot() {
```

Comment:
We planned to not limit this module to German/DE logo. While Sales Team sold this place once it's very likely we'll get such requests for other languages/conditions. This are reasons why we decided to call it `rightSlot`.